addons/website_form/views/snippets.xml

```diff
@@ -27,6 +27,10 @@
             <xpath expr="//*[@id='so_snippet_addition']" position="attributes">
                 <attribute name="data-selector" add=".s_website_form" separator=","/>
             </xpath>
+            <!-- Remove Anchor Name option from form input -->
+            <xpath expr="//div[@data-js='anchorName']" position="attributes">
```

Comment:
This commit (`[IMP] website_form: Remove anchor name option from form input`) can be removed.
The goal was to clean the UI and avoid having a customize menu with only one option but since now customize menus also contains all the parent customize option, this part can be removed.
(Plus it would have been difficult if another module would have wanted to do the same)

Reply:
Ok, Commit Removed.